### PR TITLE
chore: Bump 2.346 profile to .1 and bump 2.332 to .4

### PR DESCRIPTION
**Pull request by NotMyFault**
The PR bumps the latest LTS release to the first point release and bumps the second last LTS release to the final security release.

### CODE REVIEW

**Review by timja — APPROVED** [review]

File: sample-plugin/pom.xml [hunk 1]
```diff
@@ -515,14 +515,14 @@
...
-                <jenkins.version>2.346</jenkins.version>
+                <jenkins.version>2.346.1</jenkins.version>
             </properties>
         </profile>
         <profile>
             <id>2.332.x</id>
             <properties>
                 <bom>2.332.x</bom>
-                <jenkins.version>2.332.1</jenkins.version>
+                <jenkins.version>2.332.4</jenkins.version>
```
Comment (jglick) [inline sample-plugin/pom.xml:525]: Somehow the current doc shows 2.332.3, FYI @daniel-beck (context: https://github.com/jenkins-infra/jenkins.io/pull/4876)

**Comment by jetersen** [comment]
@NotMyFault FYI these versions are now fully managed by https://github.com/jenkinsci/bom/blob/master/updatecli/generate-manifests.ps1

I noticed however a bug after reviewing this PR 😅
Potentially a `2.446.x` can be set as `x.y` I assumed in the script it could only be `x.y.z` let me write a fix 😅

**Comment by jetersen** [comment]
#1344 should correctly handle jenkins.version for weekly or lts whether it is `x.y` or `x.y.z`

File: sample-plugin/pom.xml (hunk 1 above)
Comment (daniel-beck) [reply sample-plugin/pom.xml:525]: https://github.com/jglick/jenkins.io/blob/aed4f7fd2d70b6325d447d00588d9d86c0acd2a8/content/_layouts/corebaseline.html.haml#L13 is just https://updates.jenkins.io/tiers.json via https://github.com/jenkins-infra/jenkins.io/blob/64ed7120a01fde6d35946423e35da51e4a95fca3/scripts/fetch-external-resources#L54-L55 and it lists versions actually used as dependency only (so we can generate matching update sites).

I guess I was lazy when I set this up and didn't want to deal with Artifactory API.

By now there's half a dozen plugins that depends on 2.332.4, so the site is fixed.

**Comment by NotMyFault** [comment]
> @NotMyFault FYI these versions are now fully managed by https://github.com/jenkinsci/bom/blob/master/updatecli/generate-manifests.ps1
> 
> I noticed however a bug after reviewing this PR 😅 Potentially a `2.446.x` can be set as `x.y` I assumed in the script it could only be `x.y.z` let me write a fix 😅

Nice, I assume we no longer need to raise PRs to update existing profiles from one point release to another?

**Comment by jetersen** [comment]
> Nice, I assume we no longer need to raise PRs to update existing profiles from one point release to another?

Indeed that is the case 👏

File: sample-plugin/pom.xml (hunk 1 above)
Comment (jglick) [reply sample-plugin/pom.xml:525]: ⇒ https://github.com/jenkins-infra/jenkins.io/pull/5320

**Pull request merged**